src/VisualStudio/IntegrationTests/CSharp/CSharpIntelliSense.cs

```diff
@@ -0,0 +1,331 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Roslyn.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities.Common;
+using Roslyn.VisualStudio.Test.Utilities.Input;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpIntelliSense : AbstractEditorTests
```

Comment:
I'm going to leave this to match the original VSI test names. I think that is much stronger than the trait names -- for now at least.